src/App.tsx

```diff
@@ -1,61 +1,124 @@
 import './App.scss';
+import { FC, useState } from 'react';
+import { getNewId, findUserById, preparedTodo } from './helpers';
+import { TodoList } from './components/TodoList/TodoList';
+import usersFromServer from './api/users';
+import { FullTodo } from './types';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export const App: FC = () => {
+  const [todos, setTodos] = useState<FullTodo[]>(preparedTodo);
+
+  const [queryTitle, setQueryTitle] = useState('');
+  const [userId, setuserId] = useState(0);
+
+  const [userValueError, setuserValueError] = useState('');
+  const [titleValueError, settitleValueError] = useState('');
```

Comment:
Every new word start with capital letter
Use more descriptive names
```suggestion
  const [userError, setUserError] = useState('');
  const [titleError, setTitleError] = useState('');
```